### PR TITLE
Update dataplane version to 1.1.10

### PR DESCRIPTION
**Pull request by rahuldutta90**
## Description

Update the dataplane sdk version to 1.1.10 to include it's fixes.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by cormacpayne** [comment]
@azuresdkci add to whitelist

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@ msbuild.log
 # please do not commit any app.config files.
 app.config
 !src/ResourceManager/Profile/Commands.Profile.Test/App.config
+!src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config
```
Comment (cormacpayne) [inline .gitignore]: @rahuldutta90 I don't believe the changes to the `App.config` and `Newtonsoft.Json` that are being made in this PR are necessary -- you should be able to update the version of your SDK without having to touch the version of `Newtonsoft.Json`. Looking at the package, it still depends on 6.0.8 (or higher) for desktop, so you shouldn't need to update the `.csproj` and `.config` files outside of the SDK version itself.

Comment (rahuldutta90) [reply .gitignore]: @cormacpayne I pinged Mark about this. "I have been trying to run live unit tests for datalake powershell after almost 3 months but its failing with "Could not load file or assembly 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. " All the azure datalake sdks depend on 6.0.8 version of newtonsoft. Does any common powershell packages change to depend on 9.0.0?"

It does not allow me to run live unit tests unless I do this.

Comment (rahuldutta90) [reply .gitignore]: @cormacpayne I checked the module, it is working fine. I suspect its the tests only.

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/packages.config [hunk 2]
```diff
@@ -19,7 +19,7 @@
   <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.5.1-preview" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.Management" version="4.1.1" targetFramework="net45" />
   <package id="Moq" version="4.2.1510.2205" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
```
Comment (markcowl) [inline src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/packages.config]: This should not be necessary

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj [hunk 3]
```diff
@@ -42,6 +42,16 @@
     <ErrorReport>prompt</ErrorReport>
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.DataLake.Store">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.DataLake.Store.2.3.0-preview\lib\net452\Microsoft.Azure.Management.DataLake.Store.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```
Comment (markcowl) [inline src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj]: Why do we need this?

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```
Comment (markcowl) [inline src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config]: Why is this needed?

Comment (rahuldutta90) [reply src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/App.config]: This is for the binding redirect to 9.0.1 since my sdk depends on 6.0.8

File: src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj (hunk 3 above)
Comment (rahuldutta90) [reply src/ResourceManager/DataLakeStore/Commands.DataLakeStore.Test/Commands.DataLakeStore.Test.csproj]: merge problem. fixed.
